Formula/w/wcurl.rb

```diff
@@ -0,0 +1,22 @@
+class Wcurl < Formula
+  desc "Wrapper around curl to easily download files"
+  homepage "https://samueloph.dev/blog/announcing-wcurl-a-curl-wrapper-to-download-files/"
+  url "https://salsa.debian.org/debian/wcurl/-/archive/2024-07-07/wcurl-2024-07-07.tar.gz"
+  sha256 "5ee1d686aeef4353cb023be341f4b34401d8c6f55039cdda5d52d47cf8db4932"
+  license "curl"
+  head "https://salsa.debian.org/debian/wcurl.git", branch: "main"
+
+  depends_on "curl"
+
+  def install
+    bin.install "wcurl"
```

Comment:
I think we should replace https://salsa.debian.org/debian/wcurl/-/blob/main/wcurl#L115 with our curl path. Otherwise it'll take the first curl in path.